android-file-chooser/src/main/java/ir/sohreco/androidfilechooser/ExternalStorageNotAvailableException.java

```diff
@@ -1,7 +1,7 @@
 package ir.sohreco.androidfilechooser;
 
-public class ExternalStorageNotAvailableException extends Exception {
-    public ExternalStorageNotAvailableException() {
+class ExternalStorageNotAvailableException extends RuntimeException {
```

Comment:
What you're saying is ideal but many developers don't think about various situations. So I think that forcing the developer to handle this situation decreases the chance of their app being crashed. Thank you for your attention.